### PR TITLE
Fix Misalignment of FlowConnectorPath on page refresh

### PR DESCRIPTION
**Pull request by stevenburnell-moj**
- Fixes issue when reloading a form overview page after scrolling or when things are redrawn on browser resize.
- Change to make the detached flow section open on page load.
- Tweaks to unconnected group spacing and title content

### Tickets
https://trello.com/c/J81Q6pQe/2011-unconnected-pages-section-to-default-to-expanded-open-61-xs
https://trello.com/c/oSDI1ZMx/2248-bug-misalignment-of-flowconnectorpath-on-page-refresh-s
https://trello.com/c/HHYMcdvi/2254-unconnected-section-layout-1

### Screenshots

**(with bug/was)**
Screenshot 2022-01-21 zoomed out but after scroll page reload

**(fixed/now)**
Screenshot 2022-01-21 zoomed out but initial page load

### CODE REVIEW

File: app/javascript/src/controller_services.js [hunk 1]
```diff
@@ -562,68 +614,52 @@ function adjustOverviewHeight($overview) {
...
-      clearTimeout(scrollTimeout);
-      adjustOverviewScrollDimensions($overview, $container);
-    }, 500);
+    clearTimeout(timeout);
+    $main.css("visibility", "hidden");
+    timeout = setTimeout(function() {
+      $container.get(0).style = ""; // reset everything
+      adjustScrollDimensionsAndPosition($container);
+      $main.css("visibility", "visible");
+    }, 1500);
```
Comment (chrispymm) [inline app/javascript/src/controller_services.js:633]: 1.5s seems quite a high timeout on resize - is there a reason for it being so large?

**Review by chrispymm — APPROVED** [review]

**Review by njseeto — APPROVED** [review]
🚀

File: app/javascript/src/controller_services.js (hunk 1 above)
Comment (stevenburnell-moj) [reply app/javascript/src/controller_services.js:633]: It's an arbitrary number so figure is not based on a calculation. Reason for a slightly larger number is because this can be triggered multiple times by either user rotating a tablet device (although we're not aiming for that audience, yet) or by resizing the browser (e.g. drag the window).

Because human interaction (by intention or accident) is involved, the higher number tries to ensure things have settled down before then going ahead with an attempt to redraw the view.

In other words, the lower the number, the more chance there is of the view trying to redraw while other events are being triggered (screen still being dragged, etc.). Again, no calculation; just guesswork, hope, and experimenting to try and get a "pause... and... refresh" feel over a "redraw! redraw! redraw!!" experience.
